src/Symfony/Component/Validator/composer.json

```diff
@@ -33,6 +32,7 @@
         "symfony/expression-language": "~3.4|~4.0",
         "symfony/cache": "~3.4|~4.0",
         "symfony/property-access": "~3.4|~4.0",
+        "symfony/translation": "~3.4|~4.0",
```

Comment:
This looks wrong to me. Older version of the component do not implement the new interface. And don't we also need a conflict rule for older component versions?